CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### Other Changes
 
+- Added parameter to get API response without svg string in the result. ([#137])
```

Comment:
Please add the change message to the end of the list. Just by convention.. Thanks.